visualizations/frontend/filters/v2/TMultiSelector.vue

```diff
@@ -150,13 +150,16 @@
 				}
 				return attrs;
 			}
-		},
+        },
 		methods: {
             onVisualizationInit() {
 				const initial_value = this.getFilterValue("selected_items");
 
                 if (initial_value) {
                     this.checked = initial_value.split('|');
+                } else if (this.defaultValue) {
+                    this.checked = this.defaultValue.split('|');
+                    this.setFilterValue("selected_items", this.defaultValue);
```

Comment:
Would it make sense to `setFilterValue` and then read from it for `this.checked`? Rather than from the `defaultValue` prop?

Not sure it makes sense, but sort of like this. 

```
                if (this.defaultValue) {
                    this.setFilterValue("selected_items", this.defaultValue);
                }
                    
                this.checked =  this.getFilterValue("selected_items");
```

Please feel free to correct me if my understanding of the `checked`/`selected_items` nexus is incorrect.

Reply:
We need to read first from `filters` to give priority to URL params before setting the default value.
This will be the order: `URL query param > default-value > empty`.
That's mainly the reason we can't call `setFilterValue` if `defaultValue` is set and then read only once because we'll be giving priority to default value even if there was already a value set by the URL.

Reply:
Got it! Thanks for the explanation!